.travis.yml

```diff
@@ -112,12 +112,13 @@ jobs:
       script:
         # Run tests with coverage
         - cd $TRAVIS_BUILD_DIR
-        - python3 -m pytest --cov
+        - python3 -m pytest
+        #- python3 -m pytest --cov
```

Comment:
How come this is committed out?

Reply:
It appears to be broken in the master branch version of the build system. There is a version conflict which will require upgrading travis, which is already in place for pre-release. I figured just to temp get by for now.